cli/src/commands/auth/login.ts

```diff
@@ -1,7 +1,7 @@
 import prompts from 'prompts';
 import { createAPIKeyThroughWebUI } from '../../auth-server.js';
 import { BaseCommand } from '../../base.js';
-import { readAPIKeyFromFile, writeAPIKey } from '../../key.js';
+import { getProfile, setProfile } from '../../credentials.js';
 
 export default class Login extends BaseCommand {
   static description = 'Authenticate with Xata';
```

Comment:
Unrelated to this PR (actually error in the frontend). If key name already exists, strange error:

![image](https://user-images.githubusercontent.com/2181866/173630023-ded22c46-bf7c-4d70-bbcc-c624afb2728a.png)

Reply:
Good to know. I'll investigate 👍

Reply:
PR to fix that https://github.com/xataio/frontend-next/pull/914